src/components/Page/Eos/Eos.test.tsx

```diff
@@ -86,43 +82,6 @@ describe('Eos component', () => {
 		expect(remediationElement2).toBeInTheDocument()
 	})
 
-	test('renders found text', () => {
-		const { getByText } = render(<Eos data={mockData} />)
-		const foundTextElement = getByText('Found text example')
-		expect(foundTextElement).toBeInTheDocument()
-	})
-
-	test('renders analysis step1 1', () => {
-		const { getByText } = render(<Eos data={mockData} />)
-		expect(getByText('DATA FLOW ANALYSIS')).toBeInTheDocument()
-		const analysisStepElement1 = getByText('Sample snippet 1')
-		expect(analysisStepElement1).toBeInTheDocument()
-		const vulnerabilityElement = getByText('6:')
-		expect(vulnerabilityElement).toBeInTheDocument()
-	})
-
-	test('renders analysis snippet 2', () => {
-		const { getByText } = render(<Eos data={mockData} />)
-		expect(getByText('DATA FLOW ANALYSIS')).toBeInTheDocument()
-		const analysisStepElement2 = getByText('Sample snippet 2')
-		expect(analysisStepElement2).toBeInTheDocument()
-		const vulnerabilityElement = getByText('3:')
-		expect(vulnerabilityElement).toBeInTheDocument()
-	})
-
-	test('renders what was found', () => {
-		const { getByText } = render(<Eos data={mockData} />)
-		expect(getByText('WHAT WAS FOUND')).toBeInTheDocument()
-		const analysisStepElement2 = getByText('Found text example')
-		expect(analysisStepElement2).toBeInTheDocument()
-	})
-
-	test('renders with unknown severity when not provided', () => {
-		const { getByText } = render(<Eos data={{ ...mockData, severity: undefined }} />)
-		const severityElement = getByText('Unknown')
-		expect(severityElement).toBeInTheDocument()
-	})
-
```

Comment:
Let's add new tests to cover the visibility of InformationTabs, ContextualAnalysis and Header